unit-tests/shared/src/test/scala/javalib/util/PropertiesTest.scala

```diff
@@ -196,7 +196,10 @@ class PropertiesTest {
       val ps = new PrintStream(out)
       props.list(ps)
       ps.close()
-      assertEquals(out.toString.trim, result.trim)
+      assertEquals(
+        out.toString.replaceAll(System.lineSeparator(), "\n").trim,
```

Comment:
Is this just an artifact of the test or is it possible the parser is incorrect? I am thinking that it is consistent if it stays on one platform and can read another but testing line endings if done on one platform and tested on another can make them inconsistent.

Reply:
If we would remove adjustment of line separator the test would fail, however, this behaviour is compliant with the JVM, both end us with the same result: 
On JVM:
 ```
[error] name=alice> but was:<...isting properties --[]
[error] name=alice>, took 0.011 sec
[error]     at javalib.util.PropertiesTest.assertResult$1(PropertiesTest.scala:201)
 ``` 
 On SN: 
```
[error] name=alice> but was:<...isting properties --[]
[error] name=alice>, took 0.001 sec
[error]     at java.lang.StackTrace$.currentStackTrace(Unknown Source)
```

Reply:
Ok, thank-you.